packages/muze/src/canvas/components/message-component.js

```diff
@@ -0,0 +1,56 @@
+import { selectElement, makeElement, applyStyle } from 'muze-utils';
+import MuzeComponent from './muze-chart-component';
+
+export default class MessageComponent extends MuzeComponent {
+    constructor (params) {
+        super(params.name, params.config.dimensions, params.config.message, 0);
+        this.setParams(params);
+    }
+
+    renderMessageComponent (container) {
```

Comment:
Keep the method name as `render`. It's redundant to keep the messageComponent as part of the method name.